LoopKitUI/Views/BaseHUDView.swift

```diff
@@ -15,6 +15,10 @@ public typealias HUDViewOrderPriority = Int
     @IBOutlet weak public var caption: UILabel! {
         didSet {
             caption?.text = "—"
+            // Setting this color in code because the nib isn't being applied correctly
```

Comment:
[nit] Do we want a TODO here so we can go back and update once the nib is being applied correctly?